### ISSUE TITLE
Request: Update keyboard layout for Kinesis Advantage 360 Pro

### ISSUE THREAD

**Issue by rdtechie** [ThePrimeagen/keyboards#7]
Hi prime, I've noticed on the twitch stream you have the Kinesis Advantage 360 Pro.
I also have this keyboard since one week.

As I'm in the process to learn VIM/Real Programmers Dvorak (thank you for converting me), I like to ask you to upload your latest keyboard layout.

On the stream today you mentioned your layout is a bit different from the one in this repo, hence my request.

I would appreciate it very much!

Cheers, Richard (rdtechie)

**Comment by ThePrimeagen** [comment]
yayay.  

i'll try to reflash my keyboard and make sure its correct.

i had a whole thing going on because of a marketing model of the keyboard.

**Comment by rdtechie** [comment]
Kinesis should sent you a brand new one. Because of you I bought the keyboard so Kinesis should be happy to have such a good ambassador. ☺️

**Comment by rdtechie** [comment]
How's the reflash going?

**Comment by AlpaGit** [comment]
Any update on this ?

**Comment by marcomayer** [comment]
I too wonder how this is going :-)

**Comment by rdtechie** [comment]
@ThePrimeagen Do you have an update to share?

**Comment by M8850** [comment]
please make a video on keyboard layouts

**Comment by Hollowforge** [comment]
Hey, love your content and you've inspired me to try out your keyboard as well as many other great things such as vim. However, as someone who's only ever used qwerty and regular keyboards, I'm at a bit of a loss when it comes to working out what i should do in terms alternative keyboard layouts. I'm learning a lot of new and interesting things at the same time and taking a lot of inspiration from you (thank you!). Is there any chance you could share what your current layout is so I've got a good starting point to go off of?

**Comment by isaacgale** [comment]
Not sure if anyone here has seen this: https://github.com/ThePrimeagen/real-prog-dvorak-zmk looks like Prime has already put something here. I use QMK on a KA2 so can't test this.

**Comment by Zelinzky** [comment]
I did this if is helpful for anyone:
https://github.com/Zelinzky/Adv360-Pro-ZMK-v3/tree/Primeagen

**Comment by pinpox** [comment]
Would also be really interested to see the actual layout you use on the Kinesis specifically. Any chance we could get an image showing the keys at least?

I'm considering buying a kinesis and am unsure what layout I'll use on it.

**Comment by Zelinzky** [comment]
@pinpox https://github.com/Zelinzky/Adv360-Pro-ZMK-dvorak-qwerty/blob/Primeagen/assets/kinesis360-programmer-dvorak.png this is more or less the config I believe approximates to what the primeagen uses, dvorak and symbols distribution. 

> Would also be really interested to see the actual layout you use on the Kinesis specifically. Any chance we could get an image showing the keys at least?
> 
> I'm considering buying a kinesis and am unsure what layout I'll use on it.

**Comment by pinpox** [comment]
> @pinpox https://github.com/Zelinzky/Adv360-Pro-ZMK-dvorak-qwerty/blob/Primeagen/assets/kinesis360-programmer-dvorak.png this is more or less the config I believe approximates to what the primeagen uses, dvorak and symbols distribution.

How does Ctrl work? I only see a Fn key, does that enable the red labeled layer? If so, that would mean 3 keys are needed to prees something like "Ctrl+A"?

**Comment by Zelinzky** [comment]
> > @pinpox https://github.com/Zelinzky/Adv360-Pro-ZMK-dvorak-qwerty/blob/Primeagen/assets/kinesis360-programmer-dvorak.png this is more or less the config I believe approximates to what the primeagen uses, dvorak and symbols distribution.
> 
> 
> 
> How does Ctrl work? I only see a Fn key, does that enable the red labeled layer? If so, that would mean 3 keys are needed to prees something like "Ctrl+A"?
> 
> 

Nope, if you look at the thumb clusters you can find crtl alt and gui keys on them, if you take a look at my repo it explains that the layer configurations make use of homerow mods, this is that red labels mean that you can hold those keys to get the modifiers while tapping them to just get the normal keystroke.
